prober/http.go

```diff
@@ -173,6 +236,11 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 			Help: "Indicates if probe failed due to regex",
 		})
 
+		probeFailedDueToHeaders = prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "probe_failed_due_to_headers",
```

Comment:
probe_http_

falied_due_regex should already cover this

Reply:
> `probe_http`
Thee are several other labels that do not have the `http`: `probe_ssl_earliest_cert_expiry` and `probe_failed_due_to_regex`. Should they be changed, too? Or can I just ditch the newly introduced `probe_failed_due_to_headers` and keep the rest as is?

Reply:
They should be changed, but at least the ssl one is commonly used for alerting so we need to be a little careful so not now.

Reply:
OK, I will not touch the existing labels.

But I am not sure if representing the failed header matches with `probe_failed_due_to_regex` is the right way. It would be helpful if the user could see immediately from the metrics where the failure originated from. I do not see any drawbacks to this either (the extra disk usage would be negligible).

If you do insist, I can reuse `probe_failed_due_to_regex`. Do you insist?

Reply:
I'd suggest reusing, either that or have two more clearly named metrics.

Reply:
OK, reused existing metric. Renaming existing ones would be a more dangerous breaking change indeed.